sysl2/sysl/tests/blackbox.sysl

```diff
@@ -0,0 +1,22 @@
+import test1
+
+Project :: Integrations [title="%(epname): %(eplongname)"]:
+
+    PROJECT-E2E "End to End Integrations" [page="Solution Design Doc", blackboxes=[['SomeApp <- BarEndpoint', 'Stop at BarEndpoint']], passthrough=["anz_com", "My Todo App", "A_B"]]:
```

Comment:
`PROJECT-E2E ` is for integration diagram. You cannot use this to generate sequence diagram.